cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -123,6 +123,11 @@ func (pe *ProbeEvent) deleteBlockDeviceEvent(msg controller.EventMessage) {
 		if isGPTBasedUUIDEnabled {
 			_ = pe.deleteBlockDevice(*device, bdAPIList)
 		} else {
+			if device.DeviceAttributes.DeviceType == blockdevice.BlockDeviceTypePartition {
+				klog.Info("GPTBasedUUID disabled. skip delete block device resource for partition.")
+				isDeactivated = false
```

Comment:
@liuminjian Does setting this to false, causes a continuous loop. Seeing [this](https://github.com/openebs/node-disk-manager/pull/639/files#diff-5d08d8d71d9d5e076aa971639a22aaffc44f8184d454cf28a57766220617cc0cR143) code. I am thinking it can happen. Can you test out the changes and make sure the log flooding does not happen due to this change, when a partition gets removed.

Sorry for giving so many comments after the reviews.

Reply:
after testing, setting false will cause log flooding.so i deleted  isDeactivated = false.